src/main/java/uk/gov/hmcts/et/common/model/hmc/LocationType.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.et.common.model.hmc;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+@Getter
+public enum LocationType {
+    COURT("court"),
+    CLUSTER("cluster"),
+    REGION("region");
+
+    public final String label;
+
+    LocationType(String label) {
+        this.label = label;
+    }
+
+    public static LocationType getByLabel(String label) {
+        LocationType location = Arrays.stream(LocationType.values())
```

Comment:
Is there no way to make this a generic method? Get an enum type and return the value from it if matches the string